crypto/x509/x_pubkey.c

```diff
@@ -22,17 +22,24 @@
 #include "crypto/x509.h"
 #include <openssl/rsa.h>
 #include <openssl/dsa.h>
+#include <openssl/decoder.h>
 #include <openssl/encoder.h>
+#include <openssl/decoder.h>
```

Comment:
Twice?  I know it's a good header file but still...

Reply:
I guess I include with *enthusiasm*!

Yeahok, I'll remove one of them

Reply:
Ummmm.....did I miss something. The original header was *en*coder, and this adds *de*coder? i.e. its not the same header twice.

Reply:
That's because the conversation page only shows you two diff lines by default.  Here's what it looks like when you go look at the file changes

![foo](https://user-images.githubusercontent.com/698918/108835471-ce044b00-75cf-11eb-9774-71ac32b821e4.png)

Reply:
Anyway, it's fixed...